src/main/java/nextstep/member/filter/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,54 @@
+package nextstep.member.filter;
+
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import nextstep.member.application.JwtTokenProvider;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.security.InvalidParameterException;
+
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+    private static final String BEARER_PREFIX = "Bearer ";
+    private static final String AUTH_URL = "/members/me";
+
+    private final JwtTokenProvider jwtTokenProvider;
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        if (!request.getRequestURI().equals(AUTH_URL)) {
+            filterChain.doFilter(request, response);
+            return;
+        }
+
+        // Authorization header에서 JWT 토큰을 가져온다.
+        String token = parseJwtToken(request.getHeader(AUTHORIZATION_HEADER));
```

Comment:
```suggestion
        String token = parseJwtToken(request.getHeader(HttpHeaders.AUTHORIZATION));
```

Spring 에서 제공하는 것을 사용해도 좋을 것 같아요 😄